terraform/10-resource-group.tf

```diff
@@ -0,0 +1,5 @@
+resource "azurerm_resource_group" "hmi_apim_rg" {
+    name     = "${var.prefix}-rg-${var.environment}"
```

Comment:
As per the naming convention resource group name should end with '-rg'. so we can move '-rg' at the end

Reply:
Good catch, changed!